sql/2019/08_Security/08_43.sql

```diff
@@ -0,0 +1,11 @@
+# 08_43: HTTP vs HTTPS
+SELECT
+  client,
+  SUM(IF(url LIKE 'https://%', 1, 0))  AS https,
+  SUM(IF(url LIKE 'http://%', 1, 0))  AS http,
+  ROUND((SUM(IF(url LIKE 'https://%', 1, 0)) / COUNT(*))*100,2) AS pct_https,
+  ROUND((SUM(IF(url LIKE 'http://%', 1, 0)) / COUNT(*))*100,2) AS pct_http
+FROM
+  `httparchive.almanac.requests` 
```

Comment:
Since you're only looking at URLs, I'd recommend querying the summary_requests dataset so it's cheaper.

Reply:
done - thanks!